apps/docs/components/collapsible/Collapsible.tsx

```diff
@@ -3,7 +3,7 @@
 import { CollapseIcon, Icon } from "@/components/icon";
```

Comment:
<img width="960" alt="image" src="https://github.com/user-attachments/assets/2e78fe4e-5446-4f7e-a077-fc24c74016df">
The default inline disclosure design should have a bottom border